databuilder/databuilder/models/type_metadata.py

```diff
@@ -0,0 +1,349 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import abc
+from typing import Any, Dict, Iterator, Optional
+
+from databuilder.models.graph_node import GraphNode
+from databuilder.models.graph_relationship import GraphRelationship
+from databuilder.models.graph_serializable import GraphSerializable
+from databuilder.models.table_metadata import DescriptionMetadata
+
+
+class TypeMetadata(abc.ABC, GraphSerializable):
+    TYPE_NODE_LABEL = 'Column_Subtype'
+    TYPE_RELATION_TYPE = 'COLUMN_SUBTYPE'
```

Comment:
maybe just SUBTYPE. Since TypeMetadatas can have other child TypeMetadatas, so COLUMN_SUBTYPE will not be true after level 1.